plugin-dev/Source/SentryEditor/Private/SentrySymToolsDownloader.cpp

```diff
@@ -38,8 +39,9 @@ void FSentrySymToolsDownloader::Download(const TFunction<void(bool)>& OnComplete
 	const FString SentryCliExecPath = GetSentryCliPath();
 	const FString CliDownloadUrl = FString::Printf(TEXT("https://github.com/getsentry/sentry-cli/releases/download/%s/%s"), *GetSentryCliVersion(), *SentryCliExecName);
 
+	const FString PluginVersion = FSentryModule::Get().GetPluginVersion();
 	const FString SymUploadScriptPath = GetSymUploadScriptPath();
-	const FString SymUploadScriptDownloadUrl = FString::Printf(TEXT("https://raw.githubusercontent.com/getsentry/sentry-unreal/main/plugin-dev/Scripts/%s"), *SentrySymUploadScriptName);
+	const FString SymUploadScriptDownloadUrl = FString::Printf(TEXT("https://raw.githubusercontent.com/getsentry/sentry-unreal/refs/tags/%s/plugin-dev/Scripts/%s"), *PluginVersion, *SentrySymUploadScriptName);
```

Comment:
Ah, nice catch!🔥

Reply:
This came during the [discussion](https://github.com/getsentry/sentry-unreal/issues/880#issuecomment-2829020033) in #880.